test/distributions/test_constraints.py

```diff
@@ -41,6 +52,15 @@ def build_constraint(constraint_fn, args, is_cuda=False):
     t = torch.cuda.DoubleTensor if is_cuda else torch.DoubleTensor
     return constraint_fn(*(t(x) if isinstance(x, list) else x for x in args))
 
+@pytest.mark.parametrize('constraint_fn, result, value', EXAMPLES)
+@pytest.mark.parametrize('is_cuda', [False,
+                                     pytest.param(True, marks=pytest.mark.skipif(not TEST_CUDA,
+                                                                                 reason='CUDA not found.'))])
+def test_constraint(constraint_fn, result, value, is_cuda):
+    t = torch.cuda.DoubleTensor if is_cuda else torch.DoubleTensor
```

Comment:
I believe that these constructors are deprecated. The correct way of writing this code would be to pass a `device` to this function and to do `torch.tensor(data, dtype=torch.double, device=device)`.

Reply:
I will correct it. Thanks for your feedback!